public/app/features/alerting/unified/rule-list/RuleListPageTitle.test.tsx

```diff
@@ -45,10 +44,10 @@ function renderRuleListPageTitle() {
     writable: true,
   });
 
-  const { ...renderResult } = render(<RuleListPageTitle title="Alert rules" />);
+  const view = render(<RuleListPageTitle title="Alert rules" />);
 
   return {
-    ...renderResult,
+    ...view,
```

Comment:
Not sure why it was being desctructured and then spread back on? And changed the name as eslint rule recommends different naming

Reply:
Autocomplete issue I guess :)